Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -662,14 +662,30 @@ static void registerLogHook()
     os_log_set_hook(OS_LOG_TYPE_DEFAULT, ^(os_log_type_t type, os_log_message_t msg) {
         if (msg->buffer_sz > 1024)
             return;
-        char* messageString = os_log_copy_message_string(msg);
-        String logString = String::fromUTF8(messageString);
-        free(messageString);
 
+        String logFormat = String::fromUTF8(msg->format);
```

Comment:
I would use `CString` for all 3 of these, to avoid `fromUTF8()` and `utf8()` calls.
Can just be something like:
```
CString logFormat(msg->format);
```